extensions/ArchLinux/modules/skins/vector.less

```
@@ -108,6 +128,10 @@ body.skin-vector-2022 {
     display: block;
     float: right;
   }
```

There should be a blank line for consistency:
```suggestion
  }

```